### ISSUE TITLE
Armageddon brand on recalling isnt calculating correctly with Runesmith notable

### ISSUE THREAD

**Issue by shot040** [PathOfBuildingCommunity/PathOfBuilding#7141]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Check for support

- [X] I've checked that the calculation is supposed to be supported. If it isn't please open a feature request instead (Red text is a feature request).

### What is the value from the calculation in-game?

Value should not increase with the transfigured gem because the skill should do 80% less average hit damage per brand attached.

### What is the value from the calculation in Path of Building?

5% higher per brand attached up to two. 

### Character build code

```shell
https://pobb.in/AfJzmvfqw5VR
```

**Comment by sida-wang** [comment]
I don't believe there is an issue here. Runesmith makes it so that each brand attached to the enemy makes the enemy take 5% increased damage. So yes if you set the number of brands attached to the enemy, the DPS will go up. 

However, in order for you to trigger the 80% less damage if targeting the branded enemy that is listed on Armageddon Brand of Recall, you need to select the `Skill is targeting the Branded enemy` config option. 

![Screenshot 2023-12-23 165542](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/42928828/9b092256-2b6e-41be-8361-3913b572b9f6)
